internals/reminders.py

```diff
@@ -101,7 +102,10 @@ def build_email_tasks(
     html = render_template(body_template_path, **body_data)
     subject = subject_format % fe.name
     if is_escalated:
-      subject = f'ESCALATED: {subject}'
+      if EMAIL_SUBJECT_PREFIX in subject:
+        subject = subject.replace(EMAIL_SUBJECT_PREFIX, "Escalation request")
```

Comment:
Nit: use single quotes.

Reply:
Should have been caught by the lint! I will add it to the Thursday meeting